sql/mob_skills.sql

```diff
@@ -1163,7 +1163,7 @@ INSERT INTO `mob_skills` VALUES (1326,988,'final_retribution',1,10.0,2000,1500,4
 -- INSERT INTO `mob_skills` VALUES (1328,1072,'ink_jet',0,7.0,2000,1500,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (1329,990,'gala_macabre',1,10.0,2000,1500,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (1330,1074,'hoof_volley',0,7.0,2000,1500,4,0,0,2,0,0,0);
--- INSERT INTO `mob_skills` VALUES (1331,1075,'counterstance',0,7.0,2000,1500,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (1331,437,'counterstance_waughroon_kid',0,7.0,2000,0,4,0,0,0,0,0,0);
```

Comment:
Is there any reason this couldn't keep its original name, and then IF any other mobs happen to use counterstance (diff from Orcish counterstance) we wouldn't need multiple scripts to do essentially the same thing.

```
(00448)   704   Counterstance
(01075)  1331  Counterstance (this one)
(01173)  1429  Counterstance
```

Reply:
I don't know of other mobs doing this to compare with, but the dats say theres at least 3 so that's why I'm asking. The way you did it may be better - I don't know.